src/overlays/actors/ovl_En_Rd/z_en_rd.c

```diff
@@ -1023,8 +1027,8 @@ void EnRd_Damage(EnRd* this, PlayState* play) {
     }
 
     this->actor.world.rot.y = this->actor.yawTowardsPlayer;
-    Math_SmoothStepToS(&this->headYRotation, 0, 1, 300, 0);
-    Math_SmoothStepToS(&this->upperBodyYRotation, 0, 1, 300, 0);
+    Math_SmoothStepToS(&this->headRotY, 0, 1, 300, 0);
```

Comment:
Since these are angles the numbers should probably be hex (300 = 0x12C). There are quite a few places in this file.